chatbot/ai-studio.yaml

```diff
@@ -1,16 +1,30 @@
-application:
-  # type: language
-  # name: chatbot
-  # description: This is a LLM chatbot application
+applications:
+  type: language
+  name: chatbot
+  description: This is a LLM chatbot application
   containers:
-    - name: chatbot-example
-      contextdir: ai-application/builds
-      #image-name: quay.io/redhat-et/chatbot:latest
+    # podman run -p 7860:7860 quay.io/sallyom/model-services:llama2-7b-chatbot-arm
     - name: chatbot-model-arm
-      contextdir: model-services/builds/arm
+      contextdir: chatbot/model_service/builds/base
```

Comment:
I'm not sure how contextdir is used with desktop - will there be a 
`cd $contextdir` then
`podman build .`
or
does `contextdir` only mean "this is where the Containerfile is" and all builds happen from the root of the repository? Can you ask the desktop team?

Reply:
I'm pretty sure contextDir points to the directory where you should be to start a build - so it will be `.` or blank if we agree that builds should start from the root of the repo. In [#17](https://github.com/redhat-et/locallm/pull/17) there's a `containerfile` field added - so here the values would be:
```
contextdir: .
containerfile: chatbot/model_service/builds/base
```

Reply:
see note below also, about adding `gitrepo` and `backend` fields